### PR TITLE
Review Date

### PR DESCRIPTION
**Pull request by alexfornuto**
## Effect
PR includes the following changes:
- Adds "Last Reviewed" to page headers.

## Example
image

## Remaining Work
- [x] Technical review
- [ ] Copy Review

## Post Launch
**Do not remove** - To be completed by the docs team upon merge:
- [ ] Update Status Report
- [ ] Remove from the [project board](https://github.com/pantheon-systems/documentation/projects/14)

### CODE REVIEW

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5410 | Difference |
| --- | --- | --- | --- |
| Performance Score | `60` | `57.99999999999999` | `-2` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5410` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17300/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17300/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5410 | Difference |
| --- | --- | --- | --- |
| Performance Score | `67` | `56.99999999999999` | `-10` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5410` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17433/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17433/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5410 | Difference |
| --- | --- | --- | --- |
| Performance Score | `63` | `55.00000000000001` | `-8` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5410` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17444/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17444/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5410 | Difference |
| --- | --- | --- | --- |
| Performance Score | `49` | `56.99999999999999` | `8` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5410` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17445/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17445/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: gatsby/src/components/headerBody/index.js [hunk 1]
```diff
@@ -25,7 +26,9 @@ const HeaderBody = ({ title, subtitle, description, slug, contributors, featured
         <p className="article-subhead">
           {description}
         </p>
-
+        <p className="review-date">
+          <time datetime={lastReviewed} pubdate="pubdate">{lastReviewed}</time>
```
Comment (carolyn-shannon) [inline gatsby/src/components/headerBody/index.js]: `datetime={lastReviewed}` needs to be converted back to ISO YYYY-MM-DD. @alexfornuto please and thank you!

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5410 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.00000000000001` | `55.00000000000001` | `-1` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5410` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17459/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17459/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5410 | Difference |
| --- | --- | --- | --- |
| Performance Score | `67` | `55.00000000000001` | `-12` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5410` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17466/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17466/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5410 | Difference |
| --- | --- | --- | --- |
| Performance Score | `67` | `59` | `-8` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5410` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17467/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17467/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5410 | Difference |
| --- | --- | --- | --- |
| Performance Score | `56.99999999999999` | `56.00000000000001` | `-1` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5410` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17487/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17487/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

File: gatsby/src/components/headerBody/index.js (hunk 1 above)
Comment (alexfornuto) [reply gatsby/src/components/headerBody/index.js]: Completed in 1febf42576b11887441de87c88967ff883a96894

**Comment by pantheondocs** [comment]
Lighthouse Audit Results:

| Description | Live | pr-5410 | Difference |
| --- | --- | --- | --- |
| Performance Score | `61` | `61` | `0` |
| Best Practice Score | `93` | `93` | `0` |
| SEO Score | `96` | `96` | `0` |
| Accessibility Score | `96` | `96` | `0` |

Lighthouse Audit Reports:

- [`pr-5410` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17491/artifacts/0//lighthouse_data/lighthouse-audit-dev.html)
- [`Pantheon Docs` HTML Report](https://circleci.com/gh/pantheon-systems/documentation/17491/artifacts/0//lighthouse_data/lighthouse-audit-reference.html)

**Comment by alexfornuto** [comment]
@carolyn-shannon if everything looks :+1: on your end, I'm ready to :shipit:.

**Pull request merged**